src/common/components/play/multiplayer/LobbyStatus.js

```diff
@@ -1,21 +1,9 @@
 import * as React from 'react';
 import { arrayOf, bool, func, object, string } from 'prop-types';
-import { withStyles } from '@material-ui/core/styles';
 import Paper from '@material-ui/core/Paper';
 
-const styles = {
-  playersOnline: {
-    '& span::after': {
```

Comment:
Aww you don't like my fancy styling? Don't you keeping this in CSS is better than dynamically deciding when to insert `,&nbsp;`?

Reply:
It makes underlining the names impossible...